pyproject.toml

```diff
@@ -0,0 +1,72 @@
+[build-system]
+requires = ["setuptools >= 61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "soundata"
+dynamic = ["version"]
+description = "Python library for loading and working with sound datasets."
+readme = "README.md"
+license = {file = "LICENSE"}
+requires-python = ">=3.7"
+authors = [
+    {name = "Magdalena Fuentes", email = "mfuentes@nyu.edu"},
```

Comment:
The list of authors here does not match the list of authors you selected for the paper.